torchao/dtypes/nf4tensor.py

```diff
@@ -546,7 +580,7 @@ class LinearNF4(torch.autograd.Function):
     def forward(ctx, input: torch.Tensor, weight: NF4Tensor):
         """Save the quantized nf4 weight for backward pass"""
         ctx.nf4_weight = weight
-        return F.linear(input, weight.get_original_weight())
```

Comment:
@drisspg - So we used to dequantize for each linear call? I guess that makes sense since it's essentially weight only quant.